README.md

```diff
@@ -168,25 +168,27 @@ The cloud credentials and identity configuration
 should use the [os-client-config](https://docs.openstack.org/os-client-config/latest/) format
 and must by specified with the `--os-client-config` flag.
 
+`cacert` can be a full path to a PEM encoded file or contents of a PEM encoded file.
+
 Current list of supported options can be seen in the following example
 configuration:
 
 ```yaml
 clouds:
- default:
-   region_name: {{ openstack_region_name }}
-   identity_api_version: 3
-   identity_interface: internal
-   auth:
-     username: {{ keystone_admin_user }}
-     password: {{ keystone_admin_password }}
-     project_name: {{ keystone_admin_project }}
-     project_domain_name: 'Default'
-     project_domain_id: 'Default' // This can replace "project_domain_name"
-     user_domain_name: 'Default'
-     auth_url: {{ admin_protocol }}://{{ kolla_internal_fqdn }}:{{ keystone_admin_port }}/v3
-     cacert: |
-            ---- BEGIN CERTIFICATE ---
+  default:
+    region_name: {{ openstack_region_name }}
+    identity_api_version: 3
```

Comment:
why the indenting change? 🤔

Reply:
`cacert` was wrongly indented as a start.
Secondly as I wrote in the PR, I did the indentation change so that it's easier to see that cacert is on the same level as auth.
The current README is not consistent on the amount of space for the indentation, for example the `verify` option has 4 spaces, whereas the `auth" has 3 spaces. And those two should be on the same indentation level.

Switching over to 2 spaces for each level shows this difference more clearly.